src/test/scala/eventstore/ScavengeITest.scala

```diff
@@ -15,6 +15,7 @@ class ScavengeITest extends TestConnection {
       val probe = TestProbe()
       actor.tell(ScavengeDatabase, probe.ref)
 
+      actor ! ScavengeDatabase
```

Comment:
could you explain this?

Reply:
As there is nothing to scavenge ES replies completed instantly, this is to make the test more likely to run into getting the expected error